experimental/dds/tree/src/id-compressor/IdCompressor.ts

```diff
@@ -742,6 +745,7 @@ export class IdCompressor {
 		// If there are overrides, we must determine which cluster object (current or overflow) each belongs to and add it.
 		const overrides = ids.overrides;
 		if (overrides !== undefined) {
+			eagerFinalIdCount -= overrides.length;
```

Comment:
I think I prefer this way. If we want a total then we can just add the `eagerFinalIdCount` and `overridesCount` in the query. I'll do it this way and add a comment that clarifies that the two values are exclusive of each other.

Reply:
Sorry, I didn't mean to change the value that is emitted by the event, I just meant to change the `eagerFinalIdCount` variable itself. Because right now it is not the count of eager final IDs, despite it being named as such. You could name it `eagerFinalIdCountExceptForOverrides`, but that seems kind of silly to me when you could just have it remain as `eagerFinalIdCount` and then do the subtraction at the log site.

Put another way, a reader of this code will see `eagerFinalIdCount` declared and will think it's a variable used for IdCompressor calculations. They won't think "this is a special variable used only for telemetry, and therefore has sort of a weird meaning, and I should therefore ignore it unless I'm doing telemetry things". So you can make it clear that it is a "special telemetry thing" by giving it a different name, but I think it is cleaner to just let it actually be what someone would expect it to be, and only transform it into the "weird telemetry value" right as you send the event. Best of both worlds that way I think!

Reply:
I pushed what I think you meant by your last comment. Mind taking a look to make sure I interpreted your comment correctly?

Reply:
Yep, that's just what I was thinking!